### PR TITLE
Teams dashboard

### PR DESCRIPTION
**Pull request by whit537**
@rohitpaulk after #3412 is ready for review, you can rebase this on that and start implementing a review dashboard for teams. It can be basically the same as the fraud review dashboard. It's admin-only so it can be ugly. :)

### CODE REVIEW

**Comment by rohitpaulk** [comment]
Rebased on master.

**Comment by rohitpaulk** [comment]
Rebased on master.

File: www/new.spt [hunk 1]
```diff
@@ -14,6 +14,12 @@ if user.participant.email_address is None:
 if not user.participant.has_payout_route:
     raise Response(400, _("You must attach a bank account or PayPal to apply for a new team."))
 
+if user.participant.email_address is None:
+    raise Response(400, _("You must have a verified email address to apply for a new team."))
+
+if not user.participant.has_payout_route:
+    raise Response(400, _("You must attach a bank account or PayPal to apply for a new team."))
+
```
Comment (whit537) [inline www/new.spt]: How did we end up with duplicates here, I wonder?

**Comment by whit537** [comment]
@rohitpaulk I'm good with this for now if you're okay with those last couple commits.

**Pull request merged**